content/content-strategy/cms/cms-requirements/section10.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: text/textblock
+---
+Authors and the content they add to a CMS will impact on the accessibility of your site. Even if your CMS is ‘accessible’, the content may not be. This applies to both the frontend and backend of a CMS.
+A CMS should also help government agencies manage content to be  compliant with accessibility requirements such as [WCAG 2.0](https://www.w3.org/TR/WCAG20/).
```

Comment:
Double space between 'to be' and 'compliant'.

Reply:
Steve. Fixed

Reply:
Steve. Fixed